charts/ark-activemq/templates/secrets.yaml

```diff
@@ -1,12 +1,23 @@
-{{- if or .Values.enabled (not (hasKey .Values "enabled")) }}
+{{- if (include "arkcase.subsystem.enabled" .) }}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "common.fullname" . }}
-  labels:
-    {{- include "common.labels" . | nindent 4 }}
+  name: {{ include "common.fullname" . | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels" . | nindent 4 }}
+    {{- with .Values.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    timestamp: {{ date "20060102150405" now | quote }}
```

Comment:
"20060102150405", is this some kind of special flag value?

Reply:
It's how that specific API works (b/c that's how go date formatting works ... I had the exact same vomitive reaction you did ... but it works so ... :D )